app/_src/guides/migration-to-the-new-policies.md

```diff
@@ -477,12 +535,14 @@ becomes:
 spec:
   targetRef:
     kind: MeshGateway
-    name: edge-gateway # where this MeshGateway has `kuma.io/service: edge-gateway`
+    name: edge
```

Comment:
But name of the `MeshGateway` object in the file is `demo-app`. So should this be "demo-app"?